staging/src/k8s.io/client-go/examples/in-cluster/main.go

```diff
@@ -43,6 +44,21 @@ func main() {
 			panic(err.Error())
 		}
 		fmt.Printf("There are %d pods in the cluster\n", len(pods.Items))
+
+		// Examples for error handling:
```

Comment:
Do we really need these changes for both examples? I am fine with extending the examples, but doing everything twice looks wrong.